plugins/ata/meson.build

```diff
@@ -6,6 +6,10 @@ install_data([
   install_dir: join_paths(datadir, 'fwupd', 'quirks.d')
 )
 
+install_data(['ata.conf'],
+  install_dir:  join_paths(sysconfdir, 'fwupd')
+)
```

Comment:
I'd guess Fedora CI is going to fail since it explicitly documents individual conf files, and this needs to be added, right?

Reply:
Ack, fixed.